signinapp/proxy.py

```diff
@@ -0,0 +1,46 @@
+from flask import Blueprint, Flask, current_app, request, Response
+from flask_login import current_user, login_required
+import requests
+
+bp = Blueprint("kanboard", __name__, url_prefix="/kanboard")
+
+
+def get_user_headers():
+    return {
+        "REMOTE-USER": current_user.email,
+        "REMOTE-EMAIL": current_user.email,
+        "REMOTE-NAME": current_user.preferred_name,
+    }
+
+
+@bp.route("/<path:path>", methods=["GET", "POST"])
+@bp.route("/", methods=["GET", "POST"])
+@login_required
+def index(path=""):
+    url = current_app.config.get("PROXY_URL") + path
```

Comment:
Is `PROXY_URL` defined in the default configs?

Reply:
I guess I can set this to the kanboard URL, not sure what a good value here really is.